Tracker/tracker.lua

```diff
@@ -8958,6 +9066,10 @@ local function Main()
       tracker:setLoopToPattern()
     end
 
+    if tracker.noteNamesActive == 1 then
+      tracker:getNoteNames()

```

Comment:
Moving this to setTake would make it update when changing the item you're working on.